resources/views/PageDesign/Macros/ItemName.twig

```diff
@@ -1,17 +1,10 @@
 {% macro get( itemName, bundleType ) %}
     {% spaceless %}
     {% if bundleType == "bundle" %}
-        {% if "[BUNDLE]" in itemName %}
-            {% set itemName = itemName | replace({"[BUNDLE]": ""}) | trim %}
-        {% endif %}
-        
+        {% set itemName = services.order.removeBundlePrefix(itemName) %}
```

Comment:
Wie mein Vorschlag im IO-PR: hier wurde in der alten Logik das Ergebnis der Ersetzung noch durch `trim` gejagt, um das führende Leerzeichen zu entfernen.